indico/modules/events/editing/templates/management/filetypes.html

```diff
@@ -1,9 +1,19 @@
 {% extends 'events/editing/management/base.html' %}
 
+{% block title %}
+    {% trans t=editable_type|capitalize %} {{ t }} {% endtrans %}
+{% endblock %}
+
 {% block subtitle %}
     {% trans %}File Types{% endtrans %}
 {% endblock %}
 
 {% block content %}
     <div id="editing-filetypes" data-event-id="{{ event.id }}" data-editable-type="{{ editable_type }}"></div>
 {% endblock %}
+
+{% block back_button_url -%}
+    {%- if self.subtitle() -%}
```

Comment:
you always have one (unless there's some template inheriting from this one that empties the block), so no need for this check